themes/theme-gmd/locale/fr-FR.json

```diff
@@ -219,7 +219,12 @@
     "default_coupon_label": "Coupon",
     "coupon_was_added": "Le coupon a été ajouté à votre panier!",
     "coupons_not_supported": "Coupons non pris en charge",
```

Comment:
In French, this translates to "coupons not supported" while in the other languages show the customer something like "Add coupons and gift certificates during checkout". The naming of the key is of course a bit unclear in that case

Reply:
I've update the translation in french and dutch. I also altered some dutch translations which differed between both themes. Additionally i've reordered the locale keys in all translations. They should now have the same order as the de-DE file in the respective theme.